lib/libzfs/libzfs_diff.c

```diff
@@ -257,14 +258,34 @@ write_inuse_diffs_one(FILE *fp, differ_info_t *di, uint64_t dobj)
 	 */
 	fobjerr = get_stats_for_obj(di, di->fromsnap, dobj, fobjname,
 	    MAXPATHLEN, &fsb);
-	if (fobjerr && di->zerr != ENOENT && di->zerr != ENOTSUP)
+	if (fobjerr && di->zerr != ENOENT && di->zerr != ENOTSUP &&
+	    di->zerr != EINVAL && di->zerr != ENOENT)
```

Comment:
Looks like there's two checks for ENOENT.

Reply:
Quite right, thanks! I'm going to assume that I unthinkingly added that condition when I added ENOENT to the if a couple lines down.

Corrected.